### ISSUE TITLE
alt+f2 doesn't work with the Chinese ibus-libpinyin input method

### ISSUE THREAD

**Issue by marmistrz** [linuxmint/cinnamon#7959]
Cinnamon 3.8.9, Arch Linux, 64bit

**Issue**
Cinnamon steals the enter press event from the ibus IME

**Steps to reproduce**
1. Install ibus-libpinyin
2. enable the chinese IME
3. alt+f2
4. write `xed`
5. press enter

**Expected behaviour**
ibus should receive the enter press event, which would prompt it to leave the latin input untouched

**Actual behavior**
nemo is launched

**Comment by a-bentofreire** [comment]
I also used ibus-libpinyin, and I configured on the ibus preferences to change the key the switches between english and chinese, and I have no problems typing chinese characters on `xed`. 
I use `Desktop: Cinnamon 3.8.9 (Gtk 3.22.30) Distro: Linux Mint 19 Tara `

![xed-pinyin](https://user-images.githubusercontent.com/7064913/48105461-a8be6800-e272-11e8-95ce-fab43ddc8a10.png)

**Comment by a-bentofreire** [comment]
Yes, in that situation it also doesn't work in Linux Mint either, the solution that I use is first switch back to latin by `<shift><super>space` before `alt+f2`.
And in case I forget this extra step to switch to latin, I just press `<esc>` and do the process again.
Surely, it would be better if the command execution dialog didn't capture those keys, and allow to switch between different IME's.
